styles/temples.css

```diff
@@ -0,0 +1,72 @@
+body {
+    font-family: "Roboto", serif;
+    font-weight: 400;
+    font-style: normal
+    
```

Comment:
I suggest adding `margin: 0;` to the `body` to ensure there’s no white border around all the content.